src/components/SearchBar/SearchBar.vue

```diff
@@ -28,18 +28,24 @@
     </text-input>
     <div
       v-if="!outsideClick"
-      class="bg-white shadow overflow-y-auto max-h-56 min-w-full absolute"
+      class="bg-white shadow overflow-y-auto min-w-full absolute"
       role="results"
     >
       <div
-        v-if="searchTerm"
-        class="text-center py-4"
+        v-if="searchTerm && header"
```

Comment:
Ohhh is this right? If there's no header, we don't want to render any of the stuff inside here?

Reply:
Hmm the way I thought about it was if we didn't have the header then only the results will show up. not any of the states for 'loading', 'X results', 'no results'. I don't know if it's necessary though so open to changing it, just thought it could be useful for some search bar features.

Reply:
Ahhh I see now, I missed that header was a Boolean. Got it. You don't need `required: false` if you have a default set for the prop. Otherwise, looks good!